test/shiftregister.js

```diff
@@ -76,6 +76,11 @@ exports["ShiftRegister"] = {
     test.ok(spy.getCall(51).calledWith(4, 1)); // latch, high
     test.equals(this.shiftRegister.value, 16);
 
+    this.shiftRegister.send(0x01, 0x01);
+    test.ok(spy.getCall(52).calledWith(4, 0));
+    test.ok(shiftOutSpy.calledWith(2, 3, true, 1));
+    test.ok(spy.getCall(101).calledWith(4, 1)); // latch, high
+    test.deepEqual(this.shiftRegister.value, [1, 1]);
```

Comment:
One last request here: split these out into a new test called "sendMany" (or something like that)